com.github.phase1geo.minder.yaml

```diff
@@ -1,7 +1,7 @@
 id: com.github.phase1geo.minder
 sdk: org.freedesktop.Sdk
 runtime: org.freedesktop.Platform
-runtime-version: '19.08'
+runtime-version: '20.08'
 base: io.elementary.BaseApp
 base-version: juno-19.08
```

Comment:
The elementary base app needs to bumped first

Reply:
That's what i've thought first but wanted to try luck because of such bug report.